etc/spack/defaults/packages.yaml

```diff
@@ -41,6 +41,7 @@ packages:
       pil: [py-pillow]
       pkgconfig: [pkgconf, pkg-config]
       scalapack: [netlib-scalapack]
+      sendmail: [ssmtp]
```

Comment:
Not sure if it's worth it to make this a virtual dependency if we only have a single implementation in Spack

Reply:
`ssmtp` was the most simple for me to implement. However, it looks like the package is not maintained anymore and [debian](https://wiki.debian.org/sSMTP) and [archlinux](https://wiki.archlinux.org/index.php/SSMTP) recommend to use [msmtp](https://wiki.debian.org/msmtp) and [opensmtpd](https://wiki.archlinux.org/index.php/OpenSMTPD) instead. I guess that both have functionality similar to `ssmtp`, so I decided to introduce a virtual package because people might want to add and use one of those instead. I can remove it, of course.

Reply:
I think I understood what you meant. I've reverted `packages.yaml`.

Reply:
It turns out that a virtual package must have an entry in `packages.yaml`. I've removed the virtual package `sendmail`. Now `openpbs` simply depends on `ssmtp`.